plutus-core/plutus-ir/test/PlutusIR/Transform/Inline/Tests.hs

```diff
@@ -76,4 +76,4 @@ prop_inline biVariant =
   withMaxSuccess numTestsForPassProp $
     testPassProp
       runQuote
-      $ \tc -> inlinePassSC True tc def (def {_biSemanticsVariant = biVariant})
+      $ \tc -> inlinePassSC 0 True tc def (def {_biSemanticsVariant = biVariant})
```

Comment:
Another benefit that using the `Size` newtype gives is that this magic 0 could be less magical:
```suggestion
      $ \tc -> inlinePassSC (Size 0) True tc def (def {_biSemanticsVariant = biVariant})
```